mungegithub/mungers/submit-queue.go

```diff
@@ -499,6 +499,7 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 		if sq.BatchEnabled {
 			http.Handle("/batch", gziphandler.GzipHandler(http.HandlerFunc(sq.serveBatch)))
 		}
+		// this endpoint is useless without access to prow
 		if sq.ProwURL != "" {
 			http.Handle("/ci-status", gziphandler.GzipHandler(http.HandlerFunc(sq.serveCIStatus)))
```

Comment:
Should we still serve this page, but with empty results? I see that `ci-status` is used in the submit-queue's script.js. If the `/ci-status` path doesn't serve http will script.js work properly?